monitor/docker.go

```diff
@@ -334,23 +324,15 @@ func (d *dockerMonitor) syncContainers() error {
 
 		if err != nil {
 			log.WithFields(log.Fields{
-				"package":   "monitor",
-				"monitor":   d,
-				"error":     err,
-				"container": container,
+				"package": "monitor",
```

Comment:
We were actually logging the container.id in the past, but that could result in a Kernel panic if the container object is nil. Ideally this should never happen and be caught by the Docker event APi upstream ...